makelib/golang.mk

```diff
@@ -284,6 +290,8 @@ Go Targets:
     generate        Runs go code generation.
     fmt             Checks go source code for formatting issues.
     fmt.simplify    Format, simplify, update source files.
+    imports         Checks go source code import lines for issues.
+    imports.fix     Updates go source files to fix issues with import lines.
```

Comment:
did you want to add these targets?

Reply:
ah, i thought it was implicit in this help text that the listed targets were prefixed with `go.`, but that is not the case.  I see that there's a "Special Targets" section above that connects these targets to their `go.*` equivalents.

For consistency, I'd like to add these targets as well.